sormas-backend/src/main/java/de/symeda/sormas/backend/selfreport/SelfReportService.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2024 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.backend.selfreport;
+
+import javax.ejb.LocalBean;
+import javax.ejb.Stateless;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.From;
+import javax.persistence.criteria.Predicate;
+
+import de.symeda.sormas.api.common.DeletableEntityType;
+import de.symeda.sormas.backend.common.AbstractCoreAdoService;
+
+@Stateless
+@LocalBean
+public class SelfReportService extends AbstractCoreAdoService<SelfReport, SelfReportJoins> {
+
+	public SelfReportService() {
+		super(SelfReport.class, DeletableEntityType.SELF_REPORT);
+	}
+
+	@Override
+	protected Predicate createUserFilterInternal(CriteriaBuilder cb, CriteriaQuery cq, From<?, SelfReport> from) {
+		return null;
```

Comment:
If this method will be implemented in #13069 or #13068, we should change/delete this acceptance criteria from #13059:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/11ae65cd-60a6-49a2-8ee3-8bad3f8dd81e)

Reply:
acceptance criteria deleted